### PR TITLE
Fixed some grammatical errors and Typos

### PR DESCRIPTION
**Pull request by Gayathri-1999**
# Contributor guide pull request

Thanks for contributing to the [Microsoft Learn contributor guide](https://learn.microsoft.com/contribute/?branch=main). Please read these instructions for processing your pull request (PR). Once you've read this text, delete it and write a brief description of the changes you're proposing.

## Quality control

- [x] 1. **Successful build with no warnings**: Review the build status to make sure **all checks are green** (Succeeded).

- [x] 2. **#Sign-off**: Once the PR is finalized and ready to be merged, indicate so by typing `#sign-off` in a new comment in the PR. Signing off means the document is ready for review and can be published at any time.

## Merge and publish

- All PRs to this repository are manually reviewed and merged.
- Once all feedback on the PR is addressed, we'll merge the PR into the main branch.

## Need help?

- If you have an open PR, tag the PR reviewers in comment with your question: `@carlyrevier, @jehchow`
- Open an issue in this repo and tag the PR reviewers.

### CODE REVIEW

**Comment by Gayathri-1999** [comment]
#sign-off

**Comment by Gayathri-1999** [comment]
#sign-off

File: Contribute/content/how-to-write-quick-edits.md [hunk 1]
```diff
@@ -11,11 +11,11 @@ ms.custom: external-contributor-guide
 
 # Edit Microsoft Learn documentation in the browser
 
-Several of Microsoft's documentation sets are open source and hosted on GitHub. Not all documentation is completely open source, but many have public-facing repositories where you can suggest changes via pull requests (PRs). This open-source approach streamlines and improves communication between product engineers, content teams, and customers.
+Several of Microsoft's documentation sets are open source and hosted on GitHub. While Not all documentation is completely open source,  many have public-facing repositories where you can suggest changes via pull requests (PRs). This open-source approach streamlines and improves communication between product engineers, content teams, and customers.
```
Comment (carlyrevier) [inline Contribute/content/how-to-write-quick-edits.md:14]:
```suggestion
Several of Microsoft's documentation sets are open source and hosted on GitHub. While not all documentation sets are completely open source, many have public-facing repositories where you can suggest changes via pull requests (PRs). This open-source approach streamlines and improves communication between product engineers, content teams, and customers.
```

File: Contribute/content/how-to-write-quick-edits.md [hunk 2]
```diff
@@ -11,11 +11,11 @@ ms.custom: external-contributor-guide
...
 
-Quick edits streamline the process to report and fix small errors and omissions in documentation. Despite all efforts, small grammar and spelling errors _do_ make their way into our published documents. While you can create issues to report mistakes, it's faster and easier to create a PR to fix the issue, when the option is available.
+Quick edits facilitiate the process to report and fix small errors and omissions in documentation. Despite all efforts, small grammar and spelling errors _do_ make their way into our published documents. While you can create issues to report mistakes, it's faster and easier to create a PR to fix the issue when the option is available.
 
-We use PRs for all changes, even for contributors who have write access. Most repositories protect the default branch, so updates must be submitted as PRs.
+We utilize PRs for all changes, even for contributors who have write access. Most repositories protect the default branch, so updates must be submitted as PRs.
```
Comment (carlyrevier) [inline Contribute/content/how-to-write-quick-edits.md:18]:
```suggestion
We use PRs for all changes, even for contributors who have write access. Most repositories protect the default branch, so updates must be submitted as PRs.
```

**Review by carlyrevier — APPROVED** [review]
@Gayathri-1999 - Thanks for the updates!

**Pull request merged**